gwardia/templates/gwardia/index.html

```diff
@@ -2,27 +2,42 @@
 
 {% block title %}Panel Gwardii{% endblock %}
 
+{% load list_events %}
+
 {% block content %}
     <div class="flex flex-col items-center">
         <h1 class="text-4xl text-white font-bold mt-16 mb-8">Panel Gwardii! WIP!</h1>
         <p class="italic text-xs text-gray-500 mb-2">Kliknij w dane spotkanie, by wyświetlić jego opis</p>
-        <ul class="w-1/2" id="js-next-meetings-container">
-            {% for meeting in upcoming_five_meetings %}
-                <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer">
-                    <p class="w-full">
-                        <span class="break-all">{{ meeting.title }}</span>
-                        <span class="float-right">{{ meeting.date_of_meeting }}</span>
-                    </p>
-                    <p class="js-meeting-description border-gray-500 p-5 w-full break-all border-t-2 hidden">
-                        {{ meeting.description|linebreaksbr }}
-                    </p>
-                </li>
-            {% endfor %}
-        </ul>
+        <div  class="w-1/2" id="js-next-meetings-container">
+            <h2 class="text-2xl text-white text-center font-bold mt-8 mb-4">Nadchodzące spotkania</h2>
+            <ul>
+                {% if upcoming_meetings %}
```

Comment:
Maybe you could move this `if` to `list_events` and maybe also `h2`?

Reply:
If we want to use this as event list, not meetings list, then we shouldn't use h2 with text specific to meetings
Maybe I misunderstood you intentions and it was supposed to be meetings list tag from start